src-docs/src/views/icon/icon_example.js

```diff
@@ -213,46 +194,50 @@ export const IconExample = {
           graph color).
         </p>
       ),
-      snippet: iconColorsSnippet,
       demo: <IconColors />,
     },
+    {
+      wrapText: false,
+      text: (
+        <>
+          <EuiText>
+            <p>
+              Two-tone icons, like our app style icons, will behave similarly to
+              normal glyphs when provided a specific color by applying the color to
+              <strong>all</strong> the shapes within.
+            </p>
+          </EuiText>
+          <EuiSpacer />
+          <AppIconColors />
+        </>
+      ),
+    },
     {
       title: 'Custom SVGs',
       text: (
-        <p>
-          The <EuiCode>type</EuiCode> prop can accept a valid enum, string or
-          React SVG Element. When using a custom SVG, please make sure it sits
-          on a square canvas and preferably utilizes one of EUI&apos;s sizes
-          (16x16, 32x32...etc). For IE11 compatibility, the SVG file{' '}
-          <em>must</em> contain a <EuiCode>viewBox</EuiCode>.
-        </p>
+        <>
+          <p>
+            The <EuiCode>type</EuiCode> prop can accept a valid enum, string or
+            React SVG Element. When using a custom SVG, please make sure it sits
+            on a square canvas and preferably utilizes one of EUI&apos;s sizes
+            (16x16, 32x32...etc).
+          </p>
+          <p>
+            When using custom SVGs for simple glyphs,
+            <strong>remove all fill attributes</strong> on the SVG and utilize
```

Comment:
We're missing a space here:
<img width="325" alt="Screenshot 2021-04-13 at 10 30 20" src="https://user-images.githubusercontent.com/2750668/114530930-52279600-9c43-11eb-8535-da663f02c1a4.png">


```suggestion
            When using custom SVGs for simple glyphs,{' '}
            <strong>remove all fill attributes</strong> on the SVG and utilize
```

Reply:
Ugh, yeah VS Code was having problems auto-formatting this page with prettier for some reason. Thanks for catching these!